src/engraving/libmscore/navigate.cpp

```diff
@@ -824,6 +930,20 @@ EngravingItem* Score::prevElement()
         case ElementType::VBOX:
         case ElementType::HBOX:
         case ElementType::TBOX: {
+            auto boxChildren = toChildPairsSet(e);
+
+            for (auto child = boxChildren.rbegin(); child != boxChildren.rend(); child++) {
+                if (getSelectedElement() != child->first) {
```

Comment:
@HemantAntony, you should also store `getSelectedElement()` in a variable outside the loop here.

Reply:
Oh right, I saw it in only one place and forgot that it was in a loop